lib/sprinkler_web/live/temperature_component.ex

```diff
@@ -0,0 +1,10 @@
+defmodule SprinklerWeb.TemperatureComponent do
+  use SprinklerWeb, :live_component
+  require Logger
+
+  def render(assigns) do
+    ~L"""
+    <div>Current temperature: <%= List.last(elem(@device, 1).tmps) %></div>
```

Comment:
Here I'm showing only the last temperature data. Could be all of them for each device, though

Reply:
I think you just want to show it for one device, this should be scoped to that and then render many components like this